### PR TITLE
Navigation Revamp + My Agreements

### PR DESCRIPTION
**Pull request by gordo-labs**
Full Navigation Revamp
DefaultLayout -> LayoutLoader (Future app switcher)
AgreementsLayout as new base layer

Simplified TopBar and MobileTopbar into Web-app for now until UI components are solidified to avoid nesting on accessing logic.

Agreement list UI revamp

### CODE REVIEW

**Review by xPi2 — COMMENTED** [review]
Too much scope, some weird things on the code & we should be able to produce cleaner PRs

**Comment by xPi2** [comment]
I surrender, I can make it work and it has too many side-effects... I'm going to restart the PR from the diff component by component to be able to fix errors while they appear

File: packages/ui-components/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
-  "include": ["src", "types", "../web-app/public/animations"],
+  "include": ["src", "types", "../web-app/public/animations", "../web-app/components/TopBar.tsx", "../web-app/components/MobileNav.tsx"],
```
[the remark this answers is not in the document]
Comment (gordo-labs) [reply packages/ui-components/tsconfig.json:3]: Hmmm, I do not know what is that. Checking

**Comment by aahna-ashina** [comment]
Closing too large PR.

**Pull request closed without merging**